Makefile

```diff
@@ -27,7 +31,8 @@ docs:
 
 .PHONY: package
 deb: build docs
-	rsync -a build/* $(UPSTREAM)/
```

Comment:
Started with `package`, but thought `deb` might be better (more specific and shorter). Just forgot to change this one. (Sorry!) Fixed per 25181dd!